sklearn/metrics/_pairwise_distances_reduction/_base.pyx

```diff
@@ -116,11 +116,10 @@ cdef class PairwiseDistancesReduction64:
         by parallelizing computation on the outer loop on chunks of X
         and reduce them.
 
-        This strategy dispatches chunks of Y uniformly on threads.
-        Each thread processes all the chunks of X in turn. This strategy is
-        a sequence of embarrassingly parallel subtasks (the inner loop on Y
-        chunks) with intermediate datastructures synchronisation at each
-        iteration of the sequential outer loop on X chunks.
+        This strategy dispatches chunks of X uniformly on threads.
+        Each thread processes all the chunks of Y in turn. This strategy is
+        a sequence of embarrassingly parallel tasks (the outer loop on X
+        chunks) with no intermediate data structures synchronization at all.
```

Comment:
With the rephrasing proposed above (i.e https://github.com/scikit-learn/scikit-learn/pull/23978/files#r929593552).
```suggestion
        
        This strategy is a sequence of embarrassingly parallel tasks
        with no intermediate data structures synchronization at all.
```